bindings/python/libmata.pyx

```diff
@@ -498,14 +500,15 @@ cdef class Nfa:
         """
         self.thisptr.get().trim()
 
-    def get_digraph(self) -> Nfa:
-        """Unify transitions to create a directed graph with at most a single transition between two states.
+    def get_one_letter_aut(self) -> Type[Nfa]:
```

Comment:
Is the `Type` necessary?

Reply:
Linter seems to think so. As in, it is suggested as a better option to just `Nfa`.

Reply:
I will revert the change. Better to not have the best typing hint than to use a deprecated class.